Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx

```diff
@@ -234,71 +232,118 @@ const WizardTest = () => {
                                 Test
                             </button>
                         </div>
-                        
+
                     </div>
                     <div className="row" style={{ margin: 0 }}>
                         <div className="col">
                             {loading == 'loading' ?
-                                <LoadingIcon Show={true} Size={40} />:
-                                <Table<IChannelList>
-                                    tableStyle={{ height: '100%' }}
-                                    cols={[
-                                        { key: 'MeterName', label: 'Meter', field: 'MeterName', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
-                                        { key: 'Name', label: 'Channel', field: 'Name', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
-                                        { key: 'NumberRaised', label: 'Raised', field: 'NumberRaised', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
-                                        { key: 'TimeInAlarm', label: 'Time in Alarm (%)', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' }, content: (item) => item.TimeInAlarm.toFixed(2) + "%" },
-                                        
-                                    ]}
-                                    tableClass="table table-hover"
-                                    data={channelList}
-                                    sortKey={sort}
-                                    ascending={asc}
-                                    onSort={(d) => {
+                                <LoadingIcon Show={true} Size={40} /> :
+                                <ReactTable.Table<IChannelList>
+                                    TableStyle={{ height: '100%' }}
+                                    TableClass="table table-hover"
+                                    TheadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
+                                    TbodyStyle={{ display: 'block', overflowY: 'scroll', maxHeight: 400, width: '100%' }}
+                                    Data={channelList}
+                                    SortKey={sort}
+                                    Ascending={asc}
+                                    OnSort={(d) => {
                                         if (sort === d.colKey)
                                             setAsc(!asc)
                                         else
                                             setSort(d.colField)
                                     }}
```

Comment:
This was the only table changed with a sort function. What changed to make this work over the previous version?